enterprise/frontend/src/metabase-enterprise/collections/constants.js

```diff
@@ -0,0 +1,18 @@
+export const REGULAR_COLLECTION = {
+  type: null,
+  name: "Regular",
+  icon: "folder",
+};
+
+export const OFFICIAL_COLLECTION = {
+  type: "official",
+  name: "Official",
+  icon: "badge",
+  color: "saturated-yellow",
+};
+
+export const AUTHORITY_LEVELS = {
+  [OFFICIAL_COLLECTION.type]: OFFICIAL_COLLECTION,
+  [REGULAR_COLLECTION.type]: REGULAR_COLLECTION,
+  regular: REGULAR_COLLECTION, // just an alias
```

Comment:
This might be a bit confusing to have an alias in this authority levels object. I saw you use it to detect if a collection has the regular type. Can we maybe just use `REGULAR_COLLECTION` in these comparisons or have a utils function like `isRegularCollection`. What do you think?

Reply:
Good call, updated to use `REGULAR_COLLECTION` directly